llvm/test/tools/llvm-exegesis/RISCV/latency-by-load.s

```diff
@@ -0,0 +1,60 @@
+# RUN: llvm-exegesis -mode=latency --benchmark-phase=assemble-measured-code -mtriple=riscv64-unknown-linux-gnu --mcpu=generic -opcode-name=LD 2>&1 | FileCheck --check-prefix=TEST1 %s
+
+TEST1:      ---
+TEST1-NEXT: mode: latency
+TEST1-NEXT: key:
+TEST1-NEXT:   instructions:
+TEST1-NEXT:     - 'LD X10 X10 i_0x0'
```

Comment:
I don' think it'll always be X10, right? Could you use regex like `LD X[[REG:[0-9]+]] X[[REG]] i_0x0`?
See: https://llvm.org/docs/CommandGuide/FileCheck.html#filecheck-string-substitution-blocks

Ditto for other occurrences in this file.

Reply:
Oh I guess we're only allowing scratch memory register here right. Nvm then.